public/app/features/dashboard/components/DashboardSettings/JsonEditorSettings.tsx

```diff
@@ -21,6 +24,7 @@ export const JsonEditorSettings: React.FC<Props> = ({ dashboard }) => {
         dashboardWatcher.reloadPage();
       });
   };
+  const styles = getStyles(config.theme);
```

Comment:
The stylesFactory is deprecated so you should remove it and use the `useStyles|useStyles2` hooks instead. Also, we're moving to GrafanaThemeV2 so it would be great if you can use that instead too.
```suggestion
const styles = useStyles2(getStyles);
```